src/Integration/FilterExceptionEvent.php

```
@@ -0,0 +1,32 @@
+{
+
+    /**
+     * @param ExceptionDataBag[] $exceptions
+     */
+    public function __construct(
+        private array $exceptions
+    )
+    {
+    }
+
+    public function getExceptions(): array
```

PHPStan (2.1.15)

Method Frosh\SentryBundle\Integration\FilterExceptionEvent::getExceptions() return type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)